core/src/main/java/org/apache/calcite/runtime/CalciteResource.java

```diff
@@ -901,8 +901,6 @@
   @BaseMessage("Not a valid input for REGEXP_REPLACE: ''{0}''")
   ExInst<CalciteException> invalidInputForRegexpReplace(String value);
 
-  @BaseMessage("Illegal behavior ''{0}'' EXTRACTVALUE: document: ''{1}'', xpath expression:"
-      + " ''{2}''")
-  ExInst<CalciteException> illegalBehaviorInExtractValueFunc(String errorBehavior, String value,
-      String xpath);
+  @BaseMessage("Illegal behavior in EXTRACTVALUE: xml: ''{0}'', xpath expression: ''{1}''")
+  ExInst<CalciteException> illegalBehaviorInExtractValueFunc(String xml, String xpath);
```

Comment:
Since the exception content is not demonstrated, "Illegal behavior" may be confusing to end-uses.
Can we keep the message style same as that of Json function, i.e., "Invalid input for ..."? E.g.,
```
  @BaseMessage("Invalid input for JSON_REMOVE: document: ''{0}'', jsonpath expressions: ''{1}''")
  ExInst<CalciteException> invalidInputForJsonRemove(String value, String pathSpecs);  
```

Reply:
Done :)

Reply:
I have verified this change in my environment, it works well. Thanks!